reflection/reflection.fbs

```diff
@@ -76,14 +76,15 @@ table Field {
     offset:ushort;  // Offset into the vtable for tables, or into the struct.
     default_integer:long = 0;
     default_real:double = 0.0;
+    default_string:string;
```

Comment:
Is this needed in this PR? If we need this for something, can we separate it to a new PR?

Reply:
It adds the Default String functionality that exists in regular generators to the bfbs generator.
I thought it'd be a good idea to add it but in retrospect I agree this could be a new PR so I'm reverting that change.